src/ui/server/controllers/insurance/business/nature-of-business/index.ts

```diff
@@ -0,0 +1,112 @@
+
+const { NATURE_OF_BUSINESS_ROOT, TURNOVER_ROOT } = EXPORTER_BUSINESS_ROUTES;
+
+const { NATURE_OF_YOUR_BUSINESS: NATURE_OF_YOUR_BUSINESS_FIELDS } = FIELDS;
+
+const pageVariables = (referenceNumber: number) => ({
+  FIELDS: {
+    GOODS_OR_SERVICES: {
+      ID: GOODS_OR_SERVICES,
+      ...NATURE_OF_YOUR_BUSINESS_FIELDS[GOODS_OR_SERVICES],
+    },
+  },
+  POST_ROUTES: {
+    NATURE_OF_BUSINESS: `${INSURANCE_ROOT}/${referenceNumber}${NATURE_OF_BUSINESS_ROOT}`,
+    SAVE_AND_BACK_URL: `${INSURANCE_ROOT}/${referenceNumber}`,
+  },
+});
+
+/**
+ * gets the template for nature of business page
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.render} renders company details page with/without previously submitted details
+ */
+const get = (req: Request, res: Response) => {
+  try {
+    const { application } = res.locals;
+
+    if (!application) {
+      return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+    }
+
+    return res.render(TEMPLATE, {
+      ...insuranceCorePageVariables({
+        PAGE_CONTENT_STRINGS: NATURE_OF_YOUR_BUSINESS,
+        BACK_LINK: req.headers.referer,
+      }),
+      ...pageVariables(application.referenceNumber),
+    });
+  } catch (error) {
+    console.error('Error getting nature of business', { error });
+    return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+  }
+};
+
+/**
+ * posts nature of business page
+ * runs validation and either renders template with errors or redirects to next page
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.redirect} Turnover page with or without errors
+ */
+const post = async (req: Request, res: Response) => {
+  try {
+    const { application } = res.locals;
+
+    if (!application) {
+      return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+    }
+
+    const { body } = req;
+
+    // populate submittedValues
+    const submittedValues = {
+      [GOODS_OR_SERVICES]: body[GOODS_OR_SERVICES],
+    };
+
+    // run validation on inputs
+    const validationErrors = natureofBusinessValidation(body);
+
+    // if any errors then render template with errors
+    if (validationErrors && Object.keys(validationErrors).length) {
+      return res.render(TEMPLATE, {
+        ...insuranceCorePageVariables({
+          PAGE_CONTENT_STRINGS: NATURE_OF_YOUR_BUSINESS,
+          BACK_LINK: req.headers.referer,
+        }),
+        ...pageVariables(application.referenceNumber),
+        validationErrors,
+        submittedValues,
+      });
+    }
+
+    // if no errors, then runs save api call to db
+    // const saveResponse = await mapAndSave.companyDetails(body, application);
+
+    // if (!saveResponse) {
+    //   return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
+    // }
+
+    return res.redirect(TURNOVER_ROOT);
+  } catch (error) {
+    console.error('Error posting company details', { error });
```

Comment:
Sorry didn't spot this in other PRs/controllers for this area, we are using `err` instead of `error`